sdk/servicebus/Azure.Messaging.ServiceBus/src/Management/ServiceBusManagementClient.cs

```diff
@@ -197,12 +211,23 @@ public virtual async Task<Response<NamespaceProperties>> GetNamespacePropertiesA
         /// <exception cref="UnauthorizedAccessException">No sufficient permission to perform this operation. You should check to ensure that your <see cref="ServiceBusManagementClient"/> has the correct <see cref="TokenCredential"/> credentials to perform this operation.</exception>
         /// <exception cref="ServiceBusException.FailureReason.ServiceBusy">The server is busy. You should wait before you retry the operation.</exception>
         /// <exception cref="ServiceBusException">An internal error or an unexpected exception occured.</exception>
-        public virtual Task<Response> DeleteQueueAsync(
+        public virtual async Task<Response> DeleteQueueAsync(
```

Comment:
Wowee. Good find. Seems like there should be some kind of analyzer for this situation.

Reply:
I actually caught this due to the Diagnostic scope checks we have as part of the test framework.